src/query/procedure/mg_procedure_impl.cpp

```diff
@@ -1089,6 +1089,18 @@ mgp_error mgp_map_at(mgp_map *map, const char *key, mgp_value **result) {
       result);
 }
 
+mgp_error mgp_key_exists(mgp_map *map, const char *key, int *result) {
+  return WrapExceptions(
+      [&map, &key]() -> int {
+        auto found_it = map->items.find(key);
+        if (found_it == map->items.end()) {
+          return 0;
```

Comment:
Why not return `bool`?

Reply:
Other functions in impl.cpp that return bool in c++ return int in impl, so to be consistent with them